doc/02-Installation.md

```diff
@@ -81,9 +81,13 @@ wget https://packages.icinga.com/centos/ICINGA-release.repo -O /etc/yum.repos.d/
 
 ```bash
 apt update
-apt -y install apt-transport-https wget gnupg
+apt -y install apt-transport-https wget
 
-wget -O - https://packages.icinga.com/icinga.key | apt-key add -
```

Comment:
Ah, well, since the `apt-key add -` is removed, it's no longer added to the global trust store. The `sources.list` thus now needs the `[signed-by=...]` attribute.

Reply:
I guess, this should be revisited at the other places as well. At least the following, but I might have missed some:
- https://github.com/Icinga/packages.icinga.com/pull/45
- https://github.com/Icinga/icinga2/pull/10136

Reply:
> or, which is even recommended, place it somewhere and use the `Signed-By` option in the `sources.list` entry

That's also what https://packages.icinga.com/debian/ and https://github.com/Icinga/icinga2/blob/master/doc/02-installation.md#debian-repository- say, that was simply not adjusted correctly in this PR.

> `root@cc5096da90c6:/# gpg --enarmor < out/usr/share/keyrings/icinga-archive-keyring.gpg > /etc/apt/trusted.gpg.d/icinga-archive-keyring.asc`

By the way, that conversion shouldn't be necessary, apt should accept `*.gpg` (binary keyring) and `*.asc` (ASCII-armored keyring) files everywhere.

Reply:
> > or, which is even recommended, place it somewhere and use the `Signed-By` option in the `sources.list` entry
> 
> That's also what https://packages.icinga.com/debian/ and https://github.com/Icinga/icinga2/blob/master/doc/02-installation.md#debian-repository- say, that was simply not adjusted correctly in this PR.

Indeed. The shell monster to populate the custom `sources.list.d` file just misses it here. I just went over the other two PRs I have linked above, and they seem to be good.

Changing the `DIST=$(awk ...` block against the one from Icinga 2 with the `signed-by` option works, as I have just verified in another container.

> By the way, that conversion shouldn't be necessary, apt should accept `*.gpg` (binary keyring) and `*.asc` (ASCII-armored keyring) files everywhere.

Yes. Impatient me just stopped reading the man page after "Make sure to use the "asc" extension for ASCII armored keys" and skipped the second part ._.